main/Main.h

```diff
@@ -128,7 +123,7 @@ extern "C" {
 bool validHome(const std::string &home_path) {
   struct stat stat_result { };
   std::string sep;
-  sep += FILE_SEPARATOR;
+  sep += utils::file::FileUtils::get_separator();
```

Comment:
why +=? The variable simply could be initialized with the return value of the function. 

Moreover, looking at the code below, I would prefer removing the WIN32 ifdef, too, and making removing the leading separator from this line:
```
#define DEFAULT_NIFI_PROPERTIES_FILE "\\conf\\minifi.properties"
```

Reply:
I think the answer to why was that I didn't notice when I was extracting various commits across different branches and auto submitted the PRs via the CLI. Thanks!